src/spikeinterface/sorters/internal/spyking_circus2.py

```diff
@@ -391,7 +391,7 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
         if folder_to_delete is not None:
             shutil.rmtree(folder_to_delete)
 
-        sorting = sorting.save(folder=sorting_folder)
+        sorting = sorting.save(folder=sorting_folder, format="npz_folder")
```

Comment:
```suggestion
        # `numpy_folder doesn't release the memmap on Windows see https://github.com/SpikeInterface/spikeinterface/pull/3664
        sorting = sorting.save(folder=sorting_folder, format="npz_folder")
```

should we include a note if this gets approved so we know why? Or you think you guys will change a bunch and so the note isn't necessary?